definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -3345,6 +3401,20 @@
     "UserRole": "caseworker-divorce-courtadmin_beta",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "25/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "SUAwaitingDA",
+    "UserRole": "[PETSOLICITOR]",
```

Comment:
shouldn't this be caseworker-divorce-superuser only? Solicitors don't use this event.